src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages = Math.ceil(total / perPage);
+  const pages = Array(numberOfPages).fill(1).map((number, i) => number + i);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={classNames(
+          'page-item',
+          { disabled: currentPage === 1 },
+        )}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled="true"
```

Comment:
set correct boolean value here (and for the "next" link as well)
it'll fix tests